fabric-content-registries-v0/src/main/java/net/fabricmc/fabric/mixin/content/registry/MixinTagManager.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.content.registry;
+
+import org.spongepowered.asm.mixin.Mixin;
+
+import net.minecraft.tag.TagManager;
+
+import net.fabricmc.fabric.impl.content.registry.FuelRegistryImpl;
+
+@Mixin(targets = "net.minecraft.tag.TagManager$1")
+public abstract class MixinTagManager implements TagManager {
+	public void apply() {
```

Comment:
This is effectively an overwrite, should be an inject

Reply:
I don't think it's possible to inject here because the `apply` method is a `default` from an interface. I tried injecting there first, but injection into default interface methods is not supported yet AFAIK.

Reply:
You are targeting an anonymous class? It should be possible to inject here I think. Of course I may be wrong

Reply:
So I injected into `ClientNetworkPlayHandler` and used the `END_DATA_PACK_RELOAD` event. That removes the need for the overwrite.